filters/ThirdParty/filter_235_adblock-iran/metadata.json

```diff
@@ -7,7 +7,7 @@
   "expires": "4 days",
   "displayNumber": 2,
   "groupId": 7,
-  "subscriptionUrl": "https://gitcdn.xyz/repo/farrokhi/adblock-iran/master/filter.txt",
+  "subscriptionUrl": "https://raw.githubusercontent.com/farrokhi/adblock-iran/master/filter.txt",
```

Comment:
template.txt must be changed

Reply:
Also what about changing `trustLevel`? It seems it has not bad scriptlets.

Reply:
@Alex-302 let's open a separate PR for that

Reply:
it really has not so many scriptlet rules and they seems to be clear